k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,89 @@
+    #!/bin/bash
+
+    usage()
+    {
+        echo "Usage: bash snapshot-cleanup.sh <pv-name> <number_of_snapshots_to_delete>"
+        echo "Note: Before running this script kindly make sure there is no tmp.json file present in script execution directory"
```

Comment:
Why don't you ensure this for the user and bail out if the script is started with a "tmp.json"

Reply:
What does "block service" do? Perhaps you want to explain the ramifications of this.